CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 - Bump up logback to `1.4.0`
 - Bump up log4j2 binding to `2.18.0`
 - Fixed InvalidInputException in Eclipse while bug reporting ([#2134](https://github.com/spotbugs/spotbugs/issues/2134))
+- Fixed detector `ThrowingExceptions` by removing false positive reports, such as syntetic methods (lambdas), methods which inherited their exception specications and methods which call throwing methods ([#2040](https://github.com/spotbugs/spotbugs/issues/2040))
```

Comment:
Misspelled "synthetic" and "specifications"

Reply:
These typos got fixed in https://github.com/spotbugs/spotbugs/pull/2072/commits/642abd7b892e39d873a1e7fb552b46ae9f2bf747, so this conversation can be resolved.